admin/workflow_wiki/README.md

```diff
@@ -529,75 +529,75 @@ We will be continuing the demo of resolving [issue 431](https://github.com/umrov
           ```
```

Comment:
Why are changes to the README going in with changes for the sim?

Reply:
When using the workflow wiki, I noticed a small formatting error with one bullet so I fixed it. It's **non-functional** so I felt it didn't harm much to be in this PR. I can also move it to a separate PR if it makes you feel better.

Reply:
Works for me I just was confused.